react-theme/button.js

```diff
@@ -0,0 +1,49 @@
+import React, {Component} from 'react'
+import Theme from 'react-theme'
+
+const theme = new Theme()
+
+theme.setSource('container', () => ({
+  textAlign: 'center'
+}))
+
+theme.setSource('button', () => ({
+  backgroundColor: '#ff0000',
+  width: '320px',
+  padding: '20px',
+  borderRadius: '5px',
+  border: 'none',
+  outline: 'none',
+  active: {
```

Comment:
if you don't mind, I would ask you to remove the `active` state since it doesn't behave as the pseudo class (it stays active forever once you click on the button).